olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -36,7 +36,56 @@ namespace read {
 class VersionedLayerClientImpl;
 
 /**
- * @brief The VersionedLayerClient aimed to acquire data from OLP services.
+ * @brief The VersionedLayerClient is aimed at acquiring data from a versioned
+ * layer of the OLP. The versioned layer stores slowly-changing data that must
+ * remain logically consistent with other layers in the catalog.
+ *
+ * @note When you request a particular version of data from a versioned layer,
+ * the partition that gets returned may have a lower version number than you
+ * requested. Only those layers and partitions that are updated have their
+ * version updated to the catalog's new version number, the version of a
+ * layer or partition represents the catalog version in which the layer or
+ * partition was last updated.
+ *
+ * @note When version is not specified, the latest is used. To query the latest
+ * version of catalog \b additional request is performed.
+ *
+ * Example with version provided, which saves one network request:
+ * @code{.cpp}
+ * const std::string kCatalog =
+ * "hrn:here:data:::here-optimized-map-for-visualization-2; const std::string
+ * kLayerId = "omv-base-v2"; const auto kHRN =
```

Comment:
const std::string should go to the next line

Reply:
Done

Reply:
Done